packages/cli/src/UserManagement/email/templates/passwordReset.html

```diff
@@ -1,5 +1,10 @@
 <p>Hi {{firstName}},</p>
 <p>Somebody asked to reset your password on n8n ({{ domain }}).</p>
+{{#if isLdapUser}}
```

Comment:
Can't we display this onscreen instead of sending the email to the user?

Reply:
Yes, much better that way.